src/lwc/utilTemplateBuilder/utilTemplateBuilder.js

```diff
@@ -385,6 +392,20 @@ const setRecordValuesOnTemplate = (templateSections, fieldMappings, record) => {
     return sections;
 };
 
+/*******************************************************************************
+ * @description Method checks for page level access. Currently checks
+ * if Advanced Mapping is on from the Data Import Custom Settings and
+ * if the Gift Entry Feature Gate is turned on.
+ */
+const getPageAccess = async () => {
+    const dataImportSettings = await getDataImportSettings();
+    const giftEntryGateSettings = await getGiftEntrySettings();
+    const isAdvancedMappingOn =
+        dataImportSettings[FIELD_MAPPING_METHOD_FIELD_INFO.fieldApiName] === ADVANCED_MAPPING;
+    const isGiftEntryEnabled = giftEntryGateSettings.Enable_Gift_Entry__c;
```

Comment:
This looks like isGiftEntryEnabled is returning "undefined" in my namespaced scratch org, even though I have the setting enabled - is it possible that something is missing here (maybe namespace related? @psadie-sfdo)

Reply:
Yeah, looks like we may want to import the field info for `Enable_Gift_Entry__c` here as we're doing with the `FIELD_MAPPING_METHOD_FIELD_INFO` for the Data Import Settings.

Reply:
@ptewson-sfdo @psadie-sfdo Oh, I see. Thanks guys. Will add this change